openmetadata-ui/src/main/resources/ui/src/components/Explore/ExploreQuickFilters.tsx

```diff
@@ -109,7 +109,7 @@ const ExploreQuickFilters: FC<ExploreQuickFiltersProps> = ({
             const uniqueOptions = [
               ...new Set(
                 suggestOptions.map((op) => ({
-                  key: op._source.name,
+                  key: op._source.displayName ?? op._source.name,
```

Comment:
@aniketkatkar97 , Do we want displayName as the key?

Reply:
Yes we will be passing displayName to the API so it is necessary